### PR TITLE
Add Trainer Community meeting schedule to README

### PR DESCRIPTION
**Pull request by karenword**
Per discussion at Trainers Leadership meeting Jan 17, 2022. Also includes instructions to submit proposal via Discussions and suggest cancellations via Issues, with email alternatives.

Please double check my link formatting shortcuts? :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,40 @@ Elected members of Leadership as of March, 2021 are:
 - Arin Basu
 - Amanda Charbonneau
 - Mark Crowe
+
+## 2022 Meeting Schedule
+
+Trainer Community meetings are open only to members of the Trainer community and invited guests. Meeting times are listed on the [Community Calendar](https://carpentries.org/community/#community-events). 
+Meeting topics may be proposed by any member of the community. Meeting topics may be proposed via this repository (ideally) or by [email](mailto: instructor.training@carpentries.org). 
+To submit a proposal here, please visit the [Discussions tab](discussions) and
```
Comment (brownsarahm) [inline README.md:36]:
```suggestion
To submit a suggest a meeting topic here, please visit the [Discussions tab](discussions) and 
```
to differentiate from other use of proposals on this repo

Comment (brownsarahm) [reply README.md:36]: this link doesn't work

**Review by brownsarahm — CHANGES_REQUESTED** [review]
the discussion/issue links look for files with those names, not the appropriate sections, i htink we hav eto use full url

**Comment by marklcrowe** [comment]
> Per discussion at Trainers Leadership meeting Feb 17, 2022. 

Karen, did you mean Jan 17, 2022?

**Review by jcoliver — DISMISSED** [review]

**Review by brownsarahm — DISMISSED** [review]

**Comment by JCSzamosi** [comment]
Looking at the meeting dates, I would just like to flag March 17, which is Purim. It's not as serious a holiday as something like Yom Kippur, but many Jews do treat it as a day of rest, so if we can avoid having a meeting on that day it might be better.

**Review by brownsarahm — APPROVED** [review]

**Pull request merged**